pkg/operator/certrotation/cabundle.go

```diff
@@ -75,7 +75,15 @@ func (c CABundleConfigMap) EnsureConfigMapCABundle(ctx context.Context, signingC
 		return nil, err
 	}
 	if originalCABundleConfigMap == nil || originalCABundleConfigMap.Data == nil || !equality.Semantic.DeepEqual(originalCABundleConfigMap.Data, caBundleConfigMap.Data) {
-		c.EventRecorder.Eventf("CABundleUpdateRequired", "%q in %q requires a new cert", c.Name, c.Namespace)
+		reason := ""
+		if originalCABundleConfigMap == nil {
+			reason = "configmap doesn't exist"
+		} else if originalCABundleConfigMap.Data == nil {
+			reason = "configmap is empty"
```

Comment:
a minor suggestions:
- can we show the name and namespace of the object that does not exist for reason "configmap doesn't exist" and "configmap is empty"

Reply:
This error would be appended to the message with namespace/name: `openshift-kube-apiserver-operator node-system-admin-signer requires a new signing cert/key pair: secret doesn't exist` in `SignerUpdateRequired` event